File: CS/DXWebApplication21/MyJsonConnection.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity;

namespace DXWebApplication21 {
    public class MyJsonConnection {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string Name { get; set; }
        public string ConnectionString { get; set; }
    }

    class JsonConnectionContext : DbContext {
        public DbSet<MyJsonConnection> Connections { get; set; }
        public JsonConnectionContext() : base("ConnectionsStorage") { }
    }
}

File: VB/DXWebApplication21/MyJsonConnection.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity

Namespace DXWebApplication21
	Public Class MyJsonConnection
		<Key>
		<DatabaseGenerated(DatabaseGeneratedOption.Identity)>
		Public Property ID() As Integer
		Public Property Name() As String
		Public Property ConnectionString() As String
	End Class

	Friend Class JsonConnectionContext
		Inherits DbContext

		Public Property Connections() As DbSet(Of MyJsonConnection)
		Public Sub New()
			MyBase.New("ConnectionsStorage")
		End Sub
	End Class
End Namespace